src/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  --block-height: 300px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: calc(var(--block-height * 3));
```

Comment:
This is not a valid CSS property.  I think you wanted to set min-width so be 300px right?